### PR TITLE
Enforce use_raa even if user is raa through select_raa

### PR DESCRIPTION
**Pull request by pablothedude**
There was a bug which allowed RAA's to execute actions if
the user was made RAA through select_raa. There however still
needed to be validated if a user could act as RAA on a
certain institution by also enforcing use_raa.

Also a bug was fixed which incorrectly set the actorId in tests.
This was done through actor while actor_id should have been used.

https://www.pivotaltracker.com/story/show/170487212

### CODE REVIEW

File: tests/behat/features/fga-use-case-c.feature [hunk 1]
```diff
@@ -21,6 +21,9 @@ Feature: Use case C: Closely cooperating institutions
                 "select_raa": [
                     "institution-a.example.com",
                     "institution-d.example.com"
+                ],
+                "use_raa": [
+                    "institution-d.example.com"
```
Comment (MKodde) [inline tests/behat/features/fga-use-case-c.feature]: Reading the RFC, it seems incorrect to add use-raa for institution d here. Can you explain why this change was introduced? Idem for the use-raa on institution-d in the config below.

File: tests/behat/features/ra_grants.feature [hunk 2]
```diff
@@ -73,79 +74,79 @@ Feature: A RA(A) should only have access to certain pages
 
   # RA-management create page
   Scenario: An anonymous user can not view the ra-management create page
-    Given I visit the "management/create-ra/00000000-0000-0000-0000-000000000004" page in the RA environment
+    Given I visit the "management/create-ra/00000000-0000-4000-a000-000000000004" page in the RA environment
```
Comment (MKodde) [inline tests/behat/features/ra_grants.feature]: Out of curiosity, why did you update the UUID's?

File: tests/behat/features/fga-use-case-c.feature (hunk 1 above)
Comment (pablothedude) [reply tests/behat/features/fga-use-case-c.feature]: If an institution is selected because it's allowed in the `select_raa` config it still has to be configured to use the institutions as RA(A) based on the `use_ra(a)` config. This is done to support virtual organizations were an organization is used as an intermediate to manage organizations while not having RA(A)'s itself.

https://www.pivotaltracker.com/story/show/170487212

File: tests/behat/features/ra_grants.feature (hunk 2 above)
Comment (pablothedude) [reply tests/behat/features/ra_grants.feature]: Because the first isn't a valid UUID v4.
